_raml/apps/products/schema/patch_product_productid_customattributes.schema.json

```diff
@@ -0,0 +1,12 @@
+{
+  "$schema": "http://json-schema.org/draft-03/schema",
+  "title": "Patch Product ProductId Custom Attributes Schema",

```

Comment:
@AliceStatler Well, that's what it is :D It somehow reflects the paths of this file. Since it was done like this throughout the NOW documentation, I chose to do it this way. To be honest, I'm not sure if this is actually necessary and maybe we could change it into "Patch Custom Attributes Schema".
@mhoehn What do you think. Does anything speak against shortening the Title?

Reply:
@AliceStatler "Patch Custom Attributes Schema" is ok

Reply:
@dkluever22 Then go for it!